src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -458,38 +459,50 @@ const post911Periods = createSelector(
 
 export const servedAfter911 = formData => !!post911Periods(formData).length;
 
+export const isDisabilityPtsd = disability =>
+  disability.toLowerCase().includes(PTSD);
+
+export const hasNewPtsdDisability = formData =>
+  _.get('view:newDisabilities', formData, false) &&
+  some(_.get('newDisabilities', formData, []), item => {
+    let hasPtsd = false;
+    if (item && typeof item.condition === 'string') {
```

Comment:
I think this could use lodash's `lowerCase()` which returns an empty string if the argument is a non string